### PR TITLE
Just a few Gemfile instructions

### PR DESCRIPTION
**Pull request by lboix**
In order for a beginner to run Active Admin asap and enjoy it!

### CODE REVIEW

File: docs/0-installation.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ gem 'activeadmin'
 More accurately, it's a [Rails Engine](http://guides.rubyonrails.org/engines.html)
 that can be injected into your existing Ruby on Rails application.
 
+If you got compatible versions issue running `bundle install`, just track the current master like explained [here](https://github.com/activeadmin/activeadmin#100)
+```ruby
+gem 'activeadmin', github: 'activeadmin'
+```
+
```
Comment (lboix) [inline docs/0-installation.md]: Copy that, thanks for the feedback!

File: docs/0-installation.md [hunk 2]
```diff
@@ -21,6 +26,11 @@ rails g active_admin:install              # creates the AdminUser class
 rails g active_admin:install User         # creates / edits the class for use with Devise
 rails g active_admin:install --skip-users # skips Devise install
 ```
+If you choose to use Devise, just add it to your Gemfile :
+```ruby
+gem 'devise'
+```
+and run `bundle install`
```
Comment (lboix) [inline docs/0-installation.md]: That's a great idea, but I never used Active Admin without Devise...

Can I let the "Without Devise" paragraph in progress ?

**Comment by timoschilling** [comment]
Start the work, I will support you

**Comment by timoschilling** [comment]
I'm going to close this, but you can provide a new PR with the changes that we spoked about.

**Pull request closed without merging**